### ISSUE TITLE
Debian bullseye 0.16.0 rc0 incorrectly shows as latest stable versin

### ISSUE THREAD

**Issue by Sonic4Spuds** [inventree/InvenTree#8029]
### Deployment Method

- [X] Installer
- [ ] Docker Development
- [ ] Docker Production
- [ ] Bare metal Development
- [X] Bare metal Production
- [ ] Digital Ocean image
- [ ] Other (please provide a link `Steps to Reproduce`

### Describe the problem*

The repository shows 

inventree_0.16.0-rc0-1722955471.6937b3a2.bullseye_amd64

as the latest stable version instead of 

inventree_0.16.0-1724624163.3f2ac5ae.bullseye_amd64

### Steps to Reproduce

Install inventree from any .deb package source method on debian bullseye

### Relevant log output

```bash
apt list --upgradable -a
Listing... Done
inventree/unknown 0.16.0-rc0-1722955471.6937b3a2.bullseye amd64 [upgradable from: 0.15.7-1723011304.0d8eb2e0.bullseye]
inventree/unknown 0.16.0-rc0-1722935028.6937b3a2.bullseye amd64
inventree/unknown 0.16.0-1724624163.3f2ac5ae.bullseye amd64
inventree/unknown,now 0.15.7-1723011304.0d8eb2e0.bullseye amd64 [installed,upgradable to: 0.16.0-rc0-1722955471.6937b3a2.bullseye]
inventree/unknown 0.15.7-1722165410.0d8eb2e0.bullseye amd64
inventree/unknown 0.15.6-1721711075.23a9485e.bullseye amd64
inventree/unknown 0.15.6-1721691853.23a9485e.bullseye amd64
inventree/unknown 0.15.5-1721283164.b13cd712.bullseye amd64
inventree/unknown 0.15.5-1720768367.cc453574.bullseye amd64
inventree/unknown 0.15.4-1718619302.fa1a9da2.bullseye amd64
inventree/unknown 0.15.3-1718619232.79c43ce2.bullseye amd64
inventree/unknown 0.15.3-1718617571.d4d9aa9d.bullseye amd64
inventree/unknown 0.15.3-1716835122.d4d9aa9d.bullseye amd64
inventree/unknown 0.15.3-1716828485.d4d9aa9d.bullseye amd64
inventree/unknown 0.15.2-1716728516.ea7aa93a.bullseye amd64
inventree/unknown 0.15.1-1716272465.be3b22ce.bullseye amd64
inventree/unknown 0.15.0-1715695991.7df92aad.bullseye amd64
inventree/unknown 0.15.0-1715688760.7df92aad.bullseye amd64
inventree/unknown 0.14.5-1715333246.b32f081a.bullseye amd64
inventree/unknown 0.14.5-1714332539.b32f081a.bullseye amd64
inventree/unknown 0.14.2-1710313395.b807e476.bullseye amd64
inventree/unknown 0.14.1-1709722400.df0bb346.bullseye amd64
inventree/unknown 0.14.0-1709585071.f4fd84f5.bullseye amd64
inventree/unknown 0.14.0-1709339795.f4fd84f5.bullseye amd64
inventree/unknown 0.13.6-1709198832.0ab173d9.bullseye amd64
inventree/unknown 0.13.5-1708199043.0804ecf5.bullseye amd64
inventree/unknown 0.13.3-1705961547.e81349e7.bullseye amd64
inventree/unknown 0.13.0-1702638825.301afaa9.bullseye amd64
inventree/unknown 0.12.8-1698964278.a7487ff8.bullseye amd64
inventree/unknown 0.12.8-1698354157.a7487ff8.bullseye amd64
inventree/unknown 0.12.7-1694889278.6563b4c4.bullseye amd64
inventree/unknown 0.12.6-1693952593.942bc535.bullseye amd64
inventree/unknown 0.12.6-1693934910.942bc535.bullseye amd64
inventree/unknown 0.12.6-1693934722.942bc535.bullseye amd64
inventree/unknown 0.12.3-1691529589.1f3a49b1.bullseye amd64
inventree/unknown 0.12.3-1691474044.1f3a49b1.bullseye amd64
inventree/unknown 0.12.1-1689282363.93b44ad8.bullseye amd64
inventree/unknown 0.12.0-1688854656.1d18b487.bullseye amd64
inventree/unknown 0.12.0-1688340232.1d18b487.bullseye amd64
inventree/unknown 0.12.0-1688099007.1d18b487.bullseye amd64
inventree/unknown 0.11.2-1684994264.4868194a.bullseye amd64
inventree/unknown 0.11.0-1681740268.593a716c.bullseye amd64
inventree/unknown 0.10.0-1675773228.0ffe7861.bullseye amd64
inventree/unknown 0.9.0-1671284280.6ca23db9.bullseye amd64
inventree/unknown 0.9.0-1671146848.6ca23db9.bullseye amd64
inventree/unknown 0.8.4-1668991105.d0418676.bullseye amd64
```

**Comment by matmair** [comment]
We have released 0.16.1 - which should fix several upgrade issues. Please try updating using regular apt upgrade
Ref https://github.com/inventree/InvenTree/pull/8034

**Comment by matmair** [comment]
Duplicate of https://github.com/inventree/InvenTree/issues/8020
